src/components/Navbar/__test__/__snapshots__/Navbar.test.js.snap

```diff
@@ -150,22 +157,22 @@ exports[`renders correctly 1`] = `
       className="flex flex-col md:flex-row gap-5"
     >
       <a
```

Comment:
Link component

Reply:
it is a link but in the snapshots it recap to be a as the html bro 😅